### ISSUE TITLE
outputLanguage=es6 with --inline still contains import statements

### ISSUE THREAD

**Issue by pflannery** [google/traceur-compiler#1842]
This is the source

a.js
```js
import {b} from './b.js';
export const lib = {
  b
};
```

b.js
```js
export class b {
  constructor() {}
}
```
and running ```traceur --outputLanguage=es6 --out ./out/lib.js --inline ./lib/a.js``` outputs 
```js
export class b {
  constructor() {}
}
import {b} from './b.js';
export const lib = {b};
System.get("../lib/a.js" + '');
```
or running ```traceur --modules inline --outputLanguage=es6 --out ./out/lib.js ./lib/a.js``` outputs 
```js
export class b {
  constructor() {}
}
import {b} from './b.js';
export const lib = {b};
```

```System.get("../lib/a.js" + '');``` is referencing itself? and ```imports``` shouldn't be here either

**Comment by johnjbarton** [comment]
Just a guess: modules=inline may have rotted, we don't depend upon it and it may need some TLC.

**Comment by arv** [comment]
If `--outputLanguage=es6` is set then we use the PureES6Transformer which does not do any module transformations.

**Comment by pflannery** [comment]
I've found a new issue that I over looked for InlineES6ModuleTransformer.

`import * as bar from './foo.js'` is outputting `$traceurRuntime.exportStar` statements. When I get time I will fix this.

**Comment by pflannery** [comment]
I found that I misunderstood the cause of the issue I was having. The real cause was due to in-lining modules that contained `export * from ...`. I've attempted a fix pr see #1985
